### DISCUSSION TITLE
How to link user actions on buttons with replies?

### DISCUSSION THREAD

**Discussion by begank** [python-telegram-bot/python-telegram-bot#4187] [Q&A]
![image](https://github.com/python-telegram-bot/python-telegram-bot/assets/20349572/79afc27e-2d20-4e22-9202-91fca6e6832c)

For example,@BotFather， if a user click the "Edit Name" button and then replies with a new name, how do I know that this reply is in response to "Edit Name" and not "Edit About"?

**Comment by Poolitzer** [comment]
You could start a conversation handler
